pkg/state/envvals_loader.go

```diff
@@ -42,13 +42,7 @@ func (ld *EnvironmentValuesLoader) LoadEnvironmentValues(missingFileHandler *str
 
 		switch strOrMap := entry.(type) {
 		case string:
-			urlOrPath := strOrMap
```

Comment:
@itscaro would you mind adding some explanation for this change?

Reply:
the function ld.storage.resolveFile() resolves 
itself url or path so we do not  need do the resolution part here.

https://github.com/helmfile/helmfile/pull/565/files/877560c09cf56305adb6bcfdc612a1b760c61026#diff-9971a72e8dfdc9ad176f7a241e51ef028950bfda8ab89de9b67a5e41ef072700R45